frontend/test/metabase/query_builder/components/view/sidebars/ChartTypeSidebar.unit.spec.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { render, fireEvent, screen, within } from "@testing-library/react";
+
+import { SAMPLE_DATABASE } from "__support__/sample_database_fixture";
+
+import ChartTypeSidebar from "metabase/query_builder/components/view/sidebars/ChartTypeSidebar";
+import { color } from "metabase/lib/colors";
+
+const DATA = {
+  rows: [[1]],
+  cols: [{ base_type: "type/Integer", name: "foo", display_name: "foo" }],
+};
+
+const setup = props => {
+  const question = SAMPLE_DATABASE.question().setDisplay("gauge");
+  render(
+    <ChartTypeSidebar
+      question={question}
+      query={question.query()}
+      result={{ data: DATA }}
+      {...props}
+    />,
+  );
+};
+
+describe("ChartSettingsSidebar", () => {
+  it("should highlight the correct display type", () => {
+    setup();
+
+    //active display type
+    expect(screen.getByText("Gauge")).toHaveStyle({ color: color("brand") });
+    expect(screen.getByTestId("Gauge-button")).toHaveStyle({
+      "background-color": color("brand"),
+    });
+
+    //inactive display type
+    expect(screen.getByText("Table")).toHaveStyle({
+      color: color("text-medium"),
```

Comment:
Asserting application logic through applied elements styles is not very resilient because it will require updating specs when we simply restyle elements. Wdyt about setting aria attributes and asserting them? It would also make the UI more accessible